tests/framework/Optimizers/GradientDescent/init_sampler_allVars.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/Optimizers.ConvergeGradient</name>
```

Comment:
the name is not correct

it is 
framework/Optimizers/GradientDescent.InitSampler

Reply:
Is that the case with the init_sampler.xml as well? I have copied it from there.